FueledUtils/Combine/ObservableObjectExtensions.swift

```diff
@@ -0,0 +1,92 @@
+// Copyright © 2020, Fueled Digital Media, LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Combine
+
+extension ObservableObject where Self.ObjectWillChangePublisher == ObservableObjectPublisher {
+	// Perform a one-way link, where the receiver will listen for changes on the object and automatically trigger its `objectWillChange` publisher
+	public func link<Object: ObservableObject>(to object: Object) {
+		object.objectWillChange.subscribe(
+			AnySubscriber(
+				receiveValue: { [weak self] _ in
+					self?.objectWillChange.send()
+					return .unlimited
+				}
+			)
+		)
+	}
+
+	public func link<ObjectCollection: Collection>(to objects: ObjectCollection) where ObjectCollection.Element: ObservableObject {
+		objects.forEach(self.link(to:))
+	}
+
+	public func link<Publisher: Combine.Publisher>(to publisher: Publisher) where Publisher.Output: ObservableObject {
+		var cancellable: AnyCancellable?
+		publisher.subscribe(
+			AnySubscriber(
+				receiveValue: { [weak self] object in
+					_ = cancellable
```

Comment:
Curious why you would need this 🤔

Reply:
@notbenoit the compiler otherwise complain that `cancellable` is written to, but never read 🤷‍♂ this removes the warning. The compiler's is usually right in almost every other cases, but is not accounting for `cancellable` RAII-like behavior.
The fact that Apple has this behavior in a native framework makes me think that we might be able to flag classes to prevent such warnings in future